doc/api/fs.md

```diff
@@ -1650,6 +1650,11 @@ operations. The specific constants currently defined are described in
 ## `fs.copyFile(src, dest[, mode], callback)`
 <!-- YAML
 added: v8.5.0
+changes:
+  - version: v14.0.0
+    pr-url: https://github.com/nodejs/node/pull/27044
+    description: Changed 'mode' argument to 'flags' and imposed
```

Comment:
Two issues I see with this:

* It actually changed from `flags` to `mode`
* I don't think documenting the name change is worthwhile. All that really matters (if anything) is the change in behavior.

Reply:
oops yes typo, you're correct 🤦🏻‍♀️ 

i don't feel terribly strongly about the name change i suppose, but i think it's helpful for users who may wonder why references to a parameter have changed and does have some value